package.json

```
@@ -6,15 +6,27 @@
     "node": "22.x"
   },
   "scripts": {
+    "dev": "next dev",
```

Biome (2.4.15)

[error] The key dev was already declared.

(lint/suspicious/noDuplicateObjectKeys)